mir_connector/inorbit_mir_connector/src/connector.py

```diff
@@ -137,6 +137,27 @@ def _inorbit_command_handler(self, command_name, args, options):
             elif script_name == "set_waiting_for" and script_args[0] == "--text":
                 self._logger.info(f"Setting 'waiting for' value to {script_args[1]}")
                 self.mission_tracking.waiting_for_text = script_args[1]
+            elif script_name == "change_map":
+                if (
+                    script_args[0] == "--mir_map_id"
```

Comment:
Is this the same as a frame id ? I'm thinking if we can rename this action to "localize" and make the argument be a pose (including frame id)

What do you think ?